Whiskey/Tasks/New-WhiskeyProGetUniversalPackage.ps1

```diff
@@ -136,6 +121,31 @@ function New-WhiskeyProGetUniversalPackage
         SemVer1 = $version.SemVer1.ToString();
     } | ConvertTo-Json -Depth 1 | Set-Content -Path (Join-Path -Path $tempPackageRoot -ChildPath 'version.json')
 
+    Write-WhiskeyVerbose -Context $TaskContext -Message ('  Name:           {0}' -f $name)

```

Comment:
I think this is too much information being written. All this information is visible in the whiskey.yml file. If we transform/change what's in the whiskey.yml file into some new form/format, we should write that new form/format out. Otherwise, be silent. I believe the transformation of source path to destination path is captured by the `Write-WhiskeyInfo` call, so I think these can all be removed.